.github/workflows/deploy.yml

```
@@ -0,0 +1,35 @@
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment'
+        required: true
+        default: 'production'
+
+defaults:
+  run:
+    working-directory: ./kustomize
+
+jobs:
+  build:
+    if: ${{ github.ref == 'refs/heads/main' && github.event.inputs.environment == 'production' }}
```

このリポジトリが main ではなくて master っぽいですね
```suggestion
    if: ${{ github.ref == 'refs/heads/master' && github.event.inputs.environment == 'production' }}
```

---

見落としてたありがとう 🙏